### DISCUSSION TITLE
How do I reset the web-page demo ?

### DISCUSSION THREAD

**Discussion by viking1972** [GrapesJS/grapesjs#4007] [Q&A]
How do I reset the web-page demo ?

**Comment by ronaldohoch** [comment]
Just run `editor.DomComponents.clear()`

**Reply by viking1972** [reply]
I'm sorry if I my question was a bit vague - I meant in the online demo !

**Reply by ronaldohoch** [reply]
Works too, just open the developer tools of your browser and type the code, then, just reload and you will get the demo page again

**Reply by viking1972** [reply]
I wasn't exactly sure what you meant, however, I tried to copy & paste in the console and it worked (much to my surprise actually), so thank you very much for helping me out :o)
